<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckDatabaseDialog</class>
 <widget class="QDialog" name="CheckDatabaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datenbank überprüfen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TFHeaderWidget" name="headerWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="currentTableLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Überprüfe tfx_bereiche... (1 von 12)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tableProgressLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Gesamtfortschritt:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="tableProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="columnProgressLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Aktuelle Tabelle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="columnProgressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TFHeaderWidget</class>
   <extends>QWidget</extends>
   <header>controls/tfheaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
